lib/serialisation/BaseIO.h

```diff
@@ -138,6 +138,54 @@ namespace Grid {
     unsigned int               dimInd_{0};
   };
   
+  // Pair IO utilities /////////////////////////////////////////////////////////
+  // helper function to parse input in the format "<obj1 obj2>"
+  template <typename T1, typename T2>
```

Comment:
I thought about a regex, but this seemed to me to be a much simpler implementation, as I don't think it is possible to directly use a regex on the istream. This code does avoid the spacing issues that you mentioned, but doesn't check that you only have 2 objects between your delimiters. I could get around this by using a regex on the string I've plucked from between the `<` and `>` delimiters.